extensions/amp-riddle-quiz/0.1/amp-riddle-quiz.js

```diff
@@ -31,8 +31,8 @@ export class AmpRiddleQuiz extends AMP.BaseElement {
     /** @private {?number} */
     this.itemHeight_ = 400; //default
 
-    /** @private {?number} */
-    this.riddleId_ = null;
+    /** @private {string} */
```

Comment:
Intentional? Will data['riddleId'] return a string or a number?

Reply:
Yes, intentional. Without this change:
```js
extensions/amp-riddle-quiz/0.1/amp-riddle-quiz.js:72: ERROR - assignment to property riddleId_ of AmpRiddleQuiz
found   : string
required: (null|number)
    this.riddleId_ = userAssert(
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

extensions/amp-riddle-quiz/0.1/amp-riddle-quiz.js:89: ERROR - actual parameter 1 of encodeURIComponent does not match formal parameter
found   : (null|number)
required: string
    iframe.src = 'https://www.riddle.com/a/iframe/' + encodeURIComponent(this.riddleId_);
                                                                         ^^^^^^^^^^^^^^
```